F1/F1/Constructor/Constructor.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ConstructorModel.swift
+//  F1
+//
+//  Created by Kaitlyn Sookdhew on 2024/04/02.
+//
+
+import Foundation
+
+// MARK: - ConstructorModel
+struct ConstructorModel: Codable {
+    let constructor: ConstructorResponse
+
+    enum CodingKeys: String, CodingKey {
+        case constructor = "MRData"
+    }
+}
+
+// MARK: - ConstructorResponse
+struct ConstructorResponse: Codable {
+    let series: String
+    let url: String
+    let limit, offset, total: String
```

Comment:
any reason some of these strings are on their own line but then the rest of the other strings are on one line? Coose a standard and stick to it
 
 let series: String
    let url: String
    let limit, offset, total: String

Reply:
do the same everywhere please